tests/io_utils.go

```diff
@@ -0,0 +1,130 @@
+package tests
```

Comment:
@xpivarc missing GPL header comments in the beginning of the file

Reply:
Added.